src/SAML2/XML/md/AffiliationDescriptor.php

```diff
@@ -251,7 +253,7 @@ public function addKeyDescriptor(KeyDescriptor $keyDescriptor): void
      */
     public function toXML(DOMElement $parent): DOMElement
     {
-        Assert::notEmpty($this->AffiliateMember);
```

Comment:
Actual bug?

Reply:
Yup, this was a bug

Reply:
I'll backport it to 4.x and mark this as resolved